index.html

```diff
@@ -7,7 +7,7 @@
 
 {% include blog.html %}
 
-{% include_relative _layouts/pagehome.html content="hello world") %}
+{% include_relative _layouts/pagehome.html content='hello world') %}
```

Comment:
seems like a painful debugging process. 💭 
Hope you have a great weekend! ☀️ 🦮 ❤️

Reply:
Thanks @nelsonic  - It's over now and have a one off process I can follow. Time to watch TV whilst I have the house to myself. 

Hope you have a good weekend!

Reply:
TV! 📺 `#MentalReset` 😉 enjoy!